src/Services/InAppPurchasesService.php

```diff
@@ -22,9 +22,9 @@ public function __construct(
     ) {
     }
 
-    public function createCart(string $extensionName, string $feature, Context $context): InAppPurchaseCartStruct
+    public function createCart(string $extensionName, string $feature, string $variant, Context $context): InAppPurchaseCartStruct
     {
-        return $this->client->createInAppPurchaseCart($extensionName, $feature, $context);
+        return $this->client->createInAppPurchaseCart($extensionName, $feature, $variant, $context);
```

Comment:
why not a nullable string? Like you do in JS?

Reply:
This actually highlighted a bug that I have since fixed. Null shouldn't be accepted. There will always be a variant.

I have now fixed this https://github.com/shopware/SwagExtensionStore/pull/55/commits/0c8ff7d3b79b2844b79203a2077f96ef253428ab